pkg/securitycontext/accessors.go

```diff
@@ -46,6 +47,7 @@ type PodSecurityContextMutator interface {
 	SetRunAsUser(*int64)
 	SetRunAsGroup(*int64)
 	SetRunAsNonRoot(*bool)
+	SetSeccompProfile(*api.SeccompProfile)
```

Comment:
create new interface you can cast to that has this method and put it in `patch_accessors.go` to reduce conflicts.